examples/chainer_simple.py

```diff
@@ -73,6 +73,10 @@ def objective(trial):
     optimizer = create_optimizer(trial, model)
 
     # Dataset
+    #
+    # To reduce the overhead of loading the dataset, please refer to the FAQ entry
+    # "How to reuse the same training/test dataset across each trial run?"
+    # (https://optuna.readthedocs.io/en/stable/faq.html).
```

Comment:
Maybe this is personal preference, but I think this comment seems too long. Can we make it simple?

For instance, we can use simpler words and the URL with in-page reference as follows:
``` To reuse datasets across trials, please refer to the FAQ entry
(https://optuna.readthedocs.io/en/stable/faq.html#how-to-reuse-dataset-across-trials).```

Reply:
Basically, your proposal seems good. But I'm afraid that the fragment part of the URL may be fragile (if we changed the FAQ title a bit, the URL will be broken).

Reply:
As you mentioned, the automatically generated anchor may be broken when we edit the title.
Then, can we use sphinx's inline markup to create the anchor manually?

For instance, add an anchor `reuse-dataset` to the title line.

```
.. _reuse-dataset:

How to reuse the same training/test dataset across each trial run?
------------------------------------------------------------------
```

And then we can use the following link:
`https://optuna.readthedocs.io/en/stable/faq.html#reuse-dataset).`

Please refer to the [reference](http://www.sphinx-doc.org/es/stable/markup/inline.html#role-ref) for further details of inline markup.